src/core/stdc/stdio.d

```diff
@@ -922,6 +922,8 @@ else version( CRuntime_DigitalMars )
     pure int  feof(FILE* stream)     { return stream._flag&_IOEOF;                       }
     ///
     pure int  ferror(FILE* stream)   { return stream._flag&_IOERR;                       }
+    ///
```

Comment:
Here you go #1281.

But seriously why does this matter? It seems like an unnecessary appeal to correctness.
Considerably lowering the delay of reviews/pull requests should be a top priority, additional obstacles don't help with that.
Also from past experience of splitting work into separate but dependent pull requests, I'm ending up with a much higher review latency. Merging the shared library stuff took almost a year.

And reviewers don't easily see that a pull is blocking a huge chain of work, e.g. I can wait for ages until someone feels obliged to look at https://github.com/D-Programming-Language/dmd/pull/4650, but we're currently 3 people working on RefCounted/Unique, so it's kind of urgent b/c in a month we'll work on something else.